leshan-integration-tests/src/test/java/org/eclipse/leshan/integration/tests/IntegrationTestHelper.java

```diff
@@ -246,6 +248,12 @@ public void deregisterClient() {
 
     public void dispose() {
         deregisterClient();
+        if (null != client) {
+            client.destroy(false);
+        }
+        if (null != server) {
+            server.destroy();
+        }
```

Comment:
Using @after would cause changes in many places. Currently I started to use JUnit Rule in californium. May be such a Rule can do a better job then @before and @after.